### ISSUE TITLE
508 compliance issues (Accessibility) Zooming greater than 125% with a browser’s built-in zoom function causes card elements placed within a multi-article block to overlap

### ISSUE THREAD

**Issue by christopherwoodland** [Azure/api-management-developer-portal#1386]
## Bug description

Our client has noticed the following accessibility concerns and asked that we submit these concerns to the product team for resolution. These issues violate the 508 compliance, which the customer is bound to.  I am willing to aid in any way to bring these concerns to resolution. Please let me know if additional context or information is required to help move these concerns along.  I can be reached internally via teams or email cwoodland@microsoft.com
Thank you in advance!

1. Zooming greater than 125% with a browser’s built-in zoom function causes card elements placed within a multi-article block to overlap. Zooming to 150% or more on the API Details page causes text on the list of endpoints to be truncated with ellipses. Zooming greater than 150% on the Profile page causes text to be truncated and overlap.
![image](https://user-images.githubusercontent.com/16299417/125957396-978960b7-13d3-47be-8afe-5ca52b243faa.png)

#1383
#1384
#1385

**Comment by christopherwoodland** [comment]
@jdm4344  Perfect!  I will reference and close these issues.

#1374, #1375, #1378, #1379, #1380.
